### DISCUSSION TITLE
Baseline check for trailing whitespace

### DISCUSSION THREAD

**Discussion by rgaiacs** [frictionlessdata/frictionlessdata.io#736] [Q&A]
Consider the following CSV file named `invalid.csv`

```
id,name,extra
1,english ,
2,english,
```

Any data visualisation using the above CSV file will have `english` and `english ` (with trailing whitespace) as categories. Would be great to have a trailing whitespace as part of the baseline check as it will be more clear to learners instead of a constraint error like `The cell "english " in row at position "2" and field "name" at position "2" does not conform to a constraint: constraint "pattern" is "^[a-z].*[a-z]$"`.

**Comment by roll** [comment]
Thanks @rgaiacs!

I've created a feature request for it - https://github.com/frictionlessdata/frictionless-py/issues/1172
